db/migrate/20191004151515_create_versioned_school_certifying_officials_archives.rb

```diff
@@ -0,0 +1,15 @@
+class CreateVersionedSchoolCertifyingOfficialsArchives < ActiveRecord::Migration[4.2]
+    def up
+      safety_assured do
+        execute "create table versioned_school_certifying_officials_archives (like versioned_school_certifying_officials
+          including defaults
+          including constraints
+          including indexes
+      );"
+      end
```

Comment:
What's the rationale for wrapping this with `saftey_assured`?

Reply:
The sql I am using to create the archive doesn’t have a ActiveRecord::Migration equivalent that i’m aware of, we’ve used that sql template to create archive tables in the past. Also safety_assured is because without it strong_migrations gem will cause db:migrate to fail.